examples/ui-demo/src/hooks/7702/constants.ts

```diff
@@ -1,3 +1,3 @@
-export const ODYSSEY_EXPLORER_URL = "https://odyssey-explorer.ithaca.xyz";
+export const BASE_SEPOLIA_EXPLORER_URL = "https://base-sepolia.blockscout.com";
```

Comment:
any reason to still have this be hard-coded instead of just using `baseSepolia.baseSepolia.blockExplorers?.default`? (maybe b/c you want to use blockscout instead of basescan for some reason?)